ocs_ci/templates/ocp-deployment/busybox.yaml

```diff
@@ -0,0 +1,22 @@
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  annotations:
+    deployment.kubernetes.io/revision: '1'
+  name: test-deployment-topology-test
+  namespace: {}
+spec:
+  selector:
+    matchLabels:
+      app: test-deployment-busybox
+  template:
+    metadata:
+      labels:
+        app: test-deployment-busybox
+    spec:
+      nodeName: {}
+      containers:
+        - name: busybox
+          image: busybox
+          command: ["sleep", "360000"]
```

Comment:
No new line at the end of file

Reply:
noted, I will fix this with next PR, I proceed to develop tests in this area